superdesk/publish/publish_content.py

```diff
@@ -96,54 +96,76 @@ def get_queue_items(retries=False):
     return get_resource_service(PUBLISH_QUEUE).get(req=request, lookup=lookup)
 
 
-@celery.task(soft_time_limit=600, bind=True)
+@celery.task(soft_time_limit=60, bind=True)
 def transmit_subscriber_items(self, queue_items, subscriber):
     # Attempt to obtain a lock for transmissions to the subscriber
     lock_name = get_lock_id("Subscriber", "Transmit", subscriber)
     if not lock(lock_name, expire=610):
```

Comment:
I think lock expiry should be greater than soft_time_limit because otherwise if a transmission is blocked the after the lock will expire the same item will be selected to be transmitted again.